spoon-control-flow/src/main/java/fr/inria/controlflow/ControlFlowBuilder.java

```diff
@@ -798,7 +798,11 @@ public <T, S> void visitCtSwitchExpression(CtSwitchExpression<T, S> switchExpres
 
 	@Override
 	public void visitCtSynchronized(CtSynchronized synchro) {
+		ControlFlowNode expressionNode = new ControlFlowNode(synchro.getExpression(), result, BranchKind.STATEMENT);
```

Comment:
If the synchronization has no object (e.g. `synchronized {}`) this will likely be null and create a node with `null`. That sounds a bit weird? Could you add a test for that?